object/check.go

```diff
@@ -597,3 +617,15 @@ func CheckUserIdProviderOrigin(userIdProvider UserIdProvider) bool {
 
 	return isProviderNameEmpty != isLdapIdEmpty
 }
+
+func checkIsUserDisabled(userAttributes []*goldap.EntryAttribute) bool {
+	isDisabled := false
+
+	for _, attr := range userAttributes {
+		if attr.Name == "userAccountControl" && attr.Values[0] == "514" {
```

Comment:
Are we sure that the first item is always exist?

Reply:
in lines 270-277 we check this